config_db.go

```diff
@@ -0,0 +1,97 @@
+package evergreen
+
+import (
+	"github.com/evergreen-ci/evergreen/db"
+	"github.com/mongodb/anser/bsonutil"
+	"gopkg.in/mgo.v2/bson"
+)
+
+var (
+	Collection  = "admin"
```

Comment:
how needed is making this public? might need a better name?

Reply:
It's used in other packages in tests. I renamed it